### PR TITLE
DM-20341: update XML for ATDome and ATDomeTrajectory

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: sal_interfaces/ATDome/ATDome_Commands.xml [hunk 1]
```diff
@@ -33,12 +33,12 @@
         <Action/>
         <Value/>
         <Description>Close both shutter doors. This is the preferred way to close the shutter, because it automatically sequences the doors to prevent the main door from interfering with the dropout door.</Description>
-        <Explanation>http://sal.lsst.org</Explanation>
+        <Explanation/>
         <item>
             <EFDB_Name>ignored</EFDB_Name>
             <Description/>
```
Comment (awclemen) [inline sal_interfaces/ATDome/ATDome_Commands.xml]: https://ts-xml.lsst.io/sal_constraints_and_recommendations.html#sal-topic-and-parameter-description-and-unit-fields
>XML will have the <Units> and <Description> fields defined for each parameter in a topic

We are missing the description for the parameters

File: sal_interfaces/ATDome/ATDome_Events.xml [hunk 2]
```diff
@@ -78,13 +78,13 @@
         <EFDB_Topic>ATDome_logevent_dropoutDoorCommandedState</EFDB_Topic>
         <Alias>dropoutDoorCommandedState</Alias>
         <Description>Commanded state of the dropout shutter door.</Description>
-        <Explanation>http://sal.lsst.org</Explanation>
+        <Explanation/>
         <item>
             <EFDB_Name>commandedState</EFDB_Name>
             <Description/>
```
Comment (awclemen) [inline sal_interfaces/ATDome/ATDome_Events.xml]: Description for the parameters are missing in a few of these as well.

File: sal_interfaces/ATDomeTrajectory/ATDomeTrajectory_Events.xml [hunk 3]
```diff
@@ -7,12 +7,12 @@
         <Author/>
         <EFDB_Topic>ATDomeTrajectory_logevent_heartbeat</EFDB_Topic>
         <Alias>heartbeat</Alias>
-        <Explanation>http://sal.lsst.org</Explanation>
+        <Explanation/>
         <item>
             <EFDB_Name>heartbeat</EFDB_Name>
             <Description/>
```
Comment (awclemen) [inline sal_interfaces/ATDomeTrajectory/ATDomeTrajectory_Events.xml]: Description missing from some of the parameters

File: sal_interfaces/ATDome/ATDome_Commands.xml (hunk 1 above)
Comment (r-owen) [reply sal_interfaces/ATDome/ATDome_Commands.xml]: Please consider relaxing this rule for a topic that has one field. The description of the field is redundant with the description of the topic. That said, I hear and obey. In some cases the results are silly but in others they were an improvement. ("active" is a strange name for a field that means `communicating` but I doubt it's worth changing).

Comment (r-owen) [reply sal_interfaces/ATDome/ATDome_Commands.xml]: (On the other hand I guess that would make automatic checking a bit more complicated.)

Comment (awclemen) [reply sal_interfaces/ATDome/ATDome_Commands.xml]: I know the redundancy is kind of a pain, but the completeness is more important - plus you are right, the testing will be more problematic.   So a quick cut and paste by the developer shouldn't be too much effort.

**Comment by r-owen** [comment]
Thanks. Either Travis CI is stuck or it takes a really long time. @rbovill do you have any idea what's wrong? Please feel free to merge it yourself (but please let me know).

**Comment by tribeiro** [comment]
It’s my fault... I enabled travis after this PR was filled so github never got the notification... I’ll merge it myself if you don’t mind.

Get Outlook for iOS<https://aka.ms/o0ukef>



On Fri, Jun 28, 2019 at 6:28 PM -0700, "Russell Owen" <notifications@github.com<mailto:notifications@github.com>> wrote:


Thanks. Either Travis CI is stuck or it takes a really long time. @rbovill<https://github.com/rbovill> do you have any idea what's wrong? Please feel free to merge it yourself (but please let me know).

**Comment by r-owen** [comment]
I don't mind. I'll leave it between you and @rbovill. Of if Travis gets unstuck and I notice, I'll merge it myself. I would like to know when it is merged so I can mark the ticket as done on Jira.

**Pull request merged**